scripts/dockerfiles/polkadot/polkadot_builder.Dockerfile

```diff
@@ -1,10 +1,10 @@
 # This is the build stage for Polkadot. Here we create the binary in a temporary image.
 FROM docker.io/paritytech/ci-linux:production as builder
-
+ARG BUILD_FLAGS=""
 WORKDIR /polkadot
 COPY . /polkadot
 
-RUN cargo build --locked --release
+RUN cargo build --locked --release $BUILD_FLAGS
```

Comment:
If some changes are made there, we should also switch from --release to --profile production

I think by the way that this is not the way to do. We don't want to force the build ARGS into the image but pass them as ENV when building. I think this is possible already today by passing the ENV when running the container.

Reply:
@koenw could you explain your use case and what is currently missing ?

Reply:
@cheWe have a use case for this. In January, you added several "Cargo features", for example `[fast-runtime]` (See the [Cargo.toml](https://github.com/paritytech/polkadot/blob/f7d808eb455737b50c5bef82914ea1ef0b18f14a/Cargo.toml#L203).

Right now, these features are completely ignored in the Dockerfile. There should be an option to execute the `cargo build` with these features, for example `cargo build --locked --release --features=fast-runtime`.